main/src/main/java/cgeo/geocaching/ImageEditActivity.java

```diff
@@ -140,6 +141,21 @@ public void onCreate(final Bundle savedInstanceState) {
         }
 
         loadImage();
+
+        // check the image details
+        binding.logImgViewBtn.setOnClickListener(view -> {
+            if (image == null) {
+                Toast.makeText(view.getContext(),
```

Comment:
I assume you might silently ignore the (unlikely) situation of a null-image. (if you want you may write a log message). But IF you insist on a toast, then it's text must be internationalized and not hardcoded.